src/stats/stats.jl

```diff
@@ -58,6 +61,12 @@ function show(io :: IO, stats :: ExecutionStats)
   println("  feasibility measure: $(stats.feas_norm)")
   println("  iterations: $(stats.iter)")
   println("  elapsed time: $(stats.elapsed_time)")
+  if length(stats.solver_specific) > 0
+    println("  solver specifics:")
+    for (k,v) in stats.solver_specific
+      println("    $k: $v")
```

Comment:
I think `show` and `print` are different: `print` prints to the default stream (https://docs.julialang.org/en/stable/stdlib/io-network/#text-i-o).